docs/conf.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
```

Comment:
Minor language typo.  Changing it to the following should do the trick:
`This file only contains a selection`

Reply:
I fixed this but maybe we should send a PR and fix it in sphinx source code.